ranger/container/settings.py

```diff
@@ -117,6 +119,8 @@
     'vcs_backend_hg': ['disabled', 'local', 'enabled'],
     'vcs_backend_svn': ['disabled', 'local', 'enabled'],
     'viewmode': ['miller', 'multipane'],
+    'show_titlebar': [False, True],
```

Comment:
Remove these, they're redundant for boolean options.

Reply:
Done.